tests/performance_tests/performance_test_basic.py

```diff
@@ -236,16 +232,34 @@ def fileOpenMode(self, filePath) -> str:
             append_write = 'w'
         return append_write
 
+    def isImportantTransaction(self, transaction):
+        if self.clusterConfig.nodeosVers == "v2":
```

Comment:
Add comment that v2 history does not include `onblock`.

Reply:
Addressed: [3127329](https://github.com/AntelopeIO/leap/pull/945/commits/31273296fee664f68d6978efa136db27a0df5ac4)